sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/blob/perf/ListBlobsTest.java

```diff
@@ -25,11 +25,14 @@ public ListBlobsTest(PerfStressOptions options) {
 
     @Override
     public void run() {
-        blobContainerClient.listBlobs().forEach(b -> { });
+        blobContainerClient.listBlobs().forEach(b -> {
+        });
     }
 
     @Override
     public Mono<Void> runAsync() {
-        return blobContainerAsyncClient.listBlobs().then();
+        return blobContainerAsyncClient.listBlobs()
+            .ignoreElements()
```

Comment:
Why add `ignoreElements()`?  Is this more or less similar to the sync implementation?

Reply:
That's a good question and I'll need to dig a little more into this. `ignoreElements` actively drops onNext signals upstream propagates while `then` actively ignores the sequence from upstream. I don't know how this plays out different for the listing operation response.

Reply:
Looks like `then` is a bit of an alias for `ignoreElements` (not exactly the same but the resulting subscriber is the same). I'll go ahead and remove `ignoreElements`.